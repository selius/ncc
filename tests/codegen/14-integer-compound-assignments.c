int main()
{
	int a;
	a = 5;

	__print_int(a += 3);
	__print_int(a -= 4);
	__print_int(a *= 3);
	__print_int(a /= 5);
	__print_int(a <<= 2);
	__print_int(a %= 3);
	__print_int(a |= 5);
	__print_int(a ^= 31);
	__print_int(a &= 8);
	__print_int(a >>= 1);

	return 0;
}
