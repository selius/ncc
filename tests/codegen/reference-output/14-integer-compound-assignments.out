8
4
12
2
8
2
7
24
8
4
